src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesProjectTreeProvider.cs

```diff
@@ -560,7 +560,6 @@ public IRule GetRule(IDependency dependency, IProjectCatalogSnapshot catalogs)
             var schema = browseObjectsCatalog.GetSchema(dependency.SchemaName);
             var itemSpec = string.IsNullOrEmpty(dependency.OriginalItemSpec) ? dependency.Path : dependency.OriginalItemSpec;
             var context = ProjectPropertiesContext.GetContext(UnconfiguredProject,
```

Comment:
I see - http://index/?leftProject=Microsoft.VisualStudio.ProjectSystem&leftSymbol=xbvk3yhkvtri&file=ProjectProperties%5CContext%5CProjectPropertiesContext.cs&line=75 says file should be path to the project file and in that case suffix is empty. So not passing it at all sounds right.